pkg/session/session.go

```diff
@@ -159,6 +159,13 @@ func GetOrCreate(ctx context.Context, params *Params) (*Session, error) {
 			return s, nil
 		}
 
+		logger.V(2).Info("logout the rest session because it is inactive")
```

Comment:
so, the session may still exist, but you may not be able to get its info because you are out of connections.

the `s.TagManager` is still an instance of TagManager with a valid login/id, but by some reason like overloaded environment, etc, you may not be able to do s.TagManager.Session(ctx), which will return to you an empty instance of the session, but effectively (at least, I think) you should tell vapi/rest you want then to logout because that session seems not to be valid anymore

Reply:
Sounds good. Thx for the explanation!